Source/Python/WarpXWrappers.cpp

```diff
@@ -224,13 +232,16 @@ extern "C"
         warpx.Evolve(numsteps);
     }
 
-    void warpx_addNParticles(int speciesnumber, int lenx,
-                             amrex::ParticleReal const * x, amrex::ParticleReal const * y, amrex::ParticleReal const * z,
-                             amrex::ParticleReal const * vx, amrex::ParticleReal const * vy, amrex::ParticleReal const * vz,
-                             int nattr, amrex::ParticleReal const * attr, int uniqueparticles)
+    void warpx_addNParticles(
+        const char* char_species_name, int lenx, amrex::ParticleReal const * x,
+        amrex::ParticleReal const * y, amrex::ParticleReal const * z,
+        amrex::ParticleReal const * vx, amrex::ParticleReal const * vy,
+        amrex::ParticleReal const * vz, int nattr,
+        amrex::ParticleReal const * attr, int uniqueparticles)
```

Comment:
```suggestion
    void warpx_addNParticles(const char* char_species_name, int lenx, 
                                               amrex::ParticleReal const * x, amrex::ParticleReal const * y, amrex::ParticleReal const * z,
                                               amrex::ParticleReal const * vx, amrex::ParticleReal const * vy, amrex::ParticleReal const * vz, int nattr,
                                               amrex::ParticleReal const * attr, int uniqueparticles)
```
I think the original format the arguments were listed in is cleaner/easier to follow, and also this way, fewer lines are changed. Additionally, currently the arguments are indented to the same level as the function contents, which makes the code more difficult to read through.

Reply:
Github preview makes the suggestion look weird, but basically, I mean to keep the original 4 line structure of the function definition with the arguments `char_species_name, lenx` grouped together; `x,y,z` grouped together; `vx,vy,vz` grouped together; and `attr, uniqueparticles` grouped together.

Reply:
I hear your argument but I'll leave this as is since this way was accepted upstream. If we need to change something in this function in the future (with an upstream push) we can change the style back.